chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -58,7 +58,7 @@ def test_no_retired_repos(self):
     def test_obsolete_chroma_diagnostics(self):
         """Test that chroma-diagnostics has been obsoleted"""
         servers = config['lustre_servers']
-        addresses = [server['address'] for server in servers]
+        addresses = [server['address'] for server in servers][0:4].append(chroma_managers[0]['address'])
```

Comment:
Either use [`config_servers`](https://github.com/intel-hpdd/intel-manager-for-lustre/blob/master/chroma-manager/tests/integration/core/api_testcase_with_test_reset.py#L210) or filter out the workers using the same kind of filter that `config_servers` is using rather than hard-coding according to our particular cluster configurations.